src/EnergyPlus/WindowEquivalentLayer.cc

```diff
@@ -752,10 +752,6 @@ void EQLWindowSurfaceHeatBalance(EnergyPlusData &state,
                     if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor != -1) {
                         state.dataSurface->Surface(SurfNum).ViewFactorSkyIR = state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor;
                     }
-                    if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor != -1) {
-                        state.dataSurface->Surface(SurfNum).ViewFactorGroundIR =
-                            state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundViewFactor;
-                    }
```

Comment:
This is redundant code. The ground view factor is now obtained from a new object as show below.